pkg/kubelet/events/event.go

```diff
@@ -90,7 +90,9 @@ const (
 
 // Pod worker event reason list
 const (
-	FailedSync = "FailedSync"
+	FailedSync   = "FailedSync"
+	ProbeFailure = "ProbeFailure"
```

Comment:
not sure if this is the best grouping of event reasons. Probes are contianer-specific so maybe add to "Container event reason list" or better yet into the "Probe event reason list"